docs_user/modules/openstack-barbican_adoption.adoc

```diff
@@ -49,6 +49,7 @@ spec:
       route: {}
     template:
       databaseInstance: openstack
+      databaseAccount: barbican
       databaseUser: barbican
```

Comment:
-1: If I'm not mistaken the problem is not that `databaseAccount` is missing, it's that `databaseUser` is present.

`databaseAccount` is optional (at least in Cinder) and it has the right default.

Reply:
databaseUser being present would be ignored by the new code.

I've seen the "accountname is blank" thing happen when the newer code is run against the older CRDs that don't have the default set up, but I still dont fully understand how directives `+kubebuilder:default=barbican` fully work

Reply:
Removing databaseUser does not work, I tried that on a separate PR. (I think the Depends-On there did not work ... it should have passed but likely complained about databaseAccount instead if the Depends-On worked?)

https://logserver.rdoproject.org/62/362/1cb60f971174e01c41379f7959c56cd4a9270f3b/github-check/data-plane-adoption-osp-17-to-extracted-crc/f367945/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/barbicans.barbican.openstack.org/barbican.yaml

https://github.com/openstack-k8s-operators/data-plane-adoption/pull/362